src/main/java/pingis/services/IO.java

```diff
@@ -5,10 +5,6 @@
  */
 package pingis.services;
 
-/**
```

Comment:
Unnecessary change. Changes like removing author comment blocks should be done in a separate PR.